api/v1alpha1/oadp_types.go

```diff
@@ -339,10 +334,11 @@ type DataProtectionApplicationSpec struct {
 	//   - azurePluginImageFqin
 	//   - gcpPluginImageFqin
 	//   - csiPluginImageFqin
-	//   - dataMoverImageFqin
 	//   - resticRestoreImageFqin
 	//   - kubevirtPluginImageFqin
+	//   - operator-type
```

Comment:
Why do we need this `operator-type` key here ? What does it mean ?

Reply:
just added the options this field has. The options it has https://github.com/openshift/oadp-operator/blob/master/api/v1alpha1/oadp_types.go#L61-L73